docs/tutorials/intro.md

```diff
@@ -0,0 +1,177 @@
+# Getting Started with JupyterGIS
+
+Welcome to the first tutorial in the JupyterGIS series! This guide will introduce you to the basics of JupyterGIS and help you set up your environment, create your first map, and understand its key features.
+
+## Objectives
```

Comment:
:star_struck: https://diataxis.fr/tutorials/#show-the-learner-where-they-ll-be-going